ui/jobs/utils.ts

```diff
@@ -41,15 +75,11 @@ export class RegexesHolder {
   }
 }
 
-export function doesJobNeedMPBar(job) {
-  return Util.isCasterDpsJob(job) || Util.isHealerJob(job) || kMeleeWithMpJobs.includes(job);
-}
+export const doesJobNeedMPBar = (job: Job): boolean =>
+  Util.isCasterDpsJob(job) || Util.isHealerJob(job) || kMeleeWithMpJobs.includes(job);
 
 // Source: http://theoryjerks.akhmorning.com/guide/speed/
-export function calcGCDFromStat(bars, stat, actionDelay) {
-  // default calculates for a 2.50s recast
-  actionDelay = actionDelay || 2500;
-
+export const calcGCDFromStat = (bars: Bars, stat: number, actionDelay = 2500): number => {
```

Comment:
I wonder if this should just pass in `level` instead of `Bars`.

Reply:
no, There are some usage like `bars.speedBuffs.circleOfPower`